src/llhttp/http.ts

```diff
@@ -400,10 +400,14 @@ export class HTTP {
         n('header_value_discard_lws'));
     }
 
+    const checkContentLengthEmptiness = this.load('header_state', {
+      [HEADER_STATE.CONTENT_LENGTH]: p.error(ERROR.INVALID_CONTENT_LENGTH, 'Empty Content-Length'),
```

Comment:
Nitpick: could you move `p.error()` to a separate variable declaration?

Reply:
Sure, is this related with line length? (80)
```js
    const checkContentLengthEmptiness = this.load('header_state', {
      [HEADER_STATE.CONTENT_LENGTH]: p.error(
        ERROR.INVALID_CONTENT_LENGTH, 'Empty Content-Length'),
    }, this.setHeaderFlags(
      this.emptySpan(span.headerValue, 'header_field_start')));
```
ok? or still better new variable?

Reply:
Better to have it as a new variable, I think.